HappyAnding/HappyAnding/Extensions/SCZ+Color.swift

```diff
@@ -574,4 +574,13 @@ extension SCZColor {
         static let opacity08 = Color(hexString: "404040", opacity: 0.08)
         static let opacity04 = Color(hexString: "404040", opacity: 0.04)
     }
+    
+    ///SCZBlue 색상 사용 시 백그라운드에 white 색상 넣어줘야 의도한 색으로 나타납니다.
+    struct SCZBlue {
+        static let color = Color(hexString: "3366FF")
```

Comment:
```suggestion
        static let strong = Color(hexString: "3366FF")
```
작은 부분을 강조할 때에만 사용되는 색상이라 네이밍을 다르게 할 필요가 있습니다